com.br_automation.buoat.xddeditor.edit/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Activator: com.br_automation.buoat.xddeditor.XDD.provider.XDDEditPlugin$I
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7

```

Comment:
No it is JavaSE-1.8 because you use the List.sort Interface which does not exist in Java 7